Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -332,7 +332,6 @@ extension MapView: DelegatingMapClientDelegate {
 
         metalView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
         metalView.autoResizeDrawable = true
-        metalView.contentScaleFactor = UIScreen.main.scale
```

Comment:
I think it should it be in-sync with `MapInitOptions.mapOptions.pixelRatio`
```swift
metalView.contentScaleFactor = self.mapOptions.pixelRatio
```

So that when developer creates a map view for external display, proper scale factor will be used by both, map and metal view, for example:

```swift
MapInitOptions(..., mapOptions: MapOptions(..., pixelRatio: UIScreen.screens[1].scale))
```